middleware/authMiddleware.js

```diff
@@ -16,6 +17,7 @@ const requireAuth = (req, res, next) => {
         res.status(401).json({ error: err.message });
```

Comment:
One cool idea here, just return within if block. That way you can emit the else block and makes code look much cleaner